src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Breadcrumb extends Component {
+	render() {
+		const { history } = this.props;
+
+		return (
+			<div className="breadcrumb">
+				{history.map((item, index) => {
+					return (
+						<React.Fragment key={index}>
+							{item.isActive ? (
+								<div className="breadcrumb--object">{item.title}</div>
+							) : (
+								<a href={item.path} className="breadcrumb--object breadcrumb--link">
```

Comment:
This should be breadcrumb__link. and if you're using same style in the breadcrum__object for the link and the container it should be extend in the css not used same classNames.

Reply:
the styling is different for __link and __object in terms of font weight to highlight difference between previous path to current path